cmd/ddev/cmd/dotddev_assets/commands/host/mysqlworkbench.example

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
 ## #ddev-generated
-## Description: Run MySQLWorkbench against current db
+## Description: Run MySQL Workbench against current db
```

Comment:
I imagine your editor did this to "help:" you, but in general, try to keep PRs confined to the scope you want them confined to. If there are two unrelated issues, do two PRs.

Reply:
Roger. I wasn't sure if separate PRs for minor typos was a good thing or not, but I can certainly see the logic.

Reply:
It's no biggie. Just saying. And it's OK here.